src/main/resources/theme/chat/chat_section.html

```diff
@@ -1,4 +1,4 @@
-<div id="chat-section-{section-id}" class="chat-section">
+<div id="chat-section-{section-id}" class="chat-section chat-section-{username}">
```

Comment:
Very good; without the `chat-section-` prefix things would get messed up if users have a name of a CSS class (e.g. "foe").